### ISSUE TITLE
Make annotation of note notation to optional

### ISSUE THREAD

**Issue by kyntk** [ryokat3/vscode-qiita-markdown-preview#1]
Thank you to create Awesome tool!!!
I'm so excited to Great experience of this tool!!!

## Reference

https://qiita.com/Qiita/items/c686397e4a0f4f11683d#note---%E8%A3%9C%E8%B6%B3%E8%AA%AC%E6%98%8E

## What

- Make pass validation when `words.length === 1 && words[0] === 'note'`

https://github.com/ryokat3/vscode-qiita-markdown-preview/blob/main/src/qiita-note.ts#L29

```md
:::note
This is also note notation.
:::
```

**Comment by ryokat3** [comment]
Hi @kyntk,

Thank you for your contribution !!

I fully accepted your suggestion and released 0.2.1 with improved 'note' notation validation and rendering.
I hope the updated version works as you expect.

**Comment by kyntk** [comment]
Hi @ryokat3 ,

Thank you for quick response!!!

However, I think it is not enough because we can write something like the following.
```md
:::notehoge
The first word is allowed starting with 'note' but not ending after that.
:::
```

**Comment by ryokat3** [comment]
Hi @kyntk,

Thank you for your report. The case you pointed out had not come to my mind at all... I think it's fixed in Release 0.2.2.

**Comment by kyntk** [comment]
Hi @ryokat3 ,

Thank you for fixing it!
I tried it and it looks good to me, so I'm going to close this issue.
